apps/docs/src/medusa-forms/ControlledCheckbox.stories.tsx

```
@@ -0,0 +1,396 @@
+    mode: 'onChange',
+  });
+
+  const onSubmit = (data: any) => {
+    alert(`Form submitted with data: ${JSON.stringify(data, null, 2)}`);
+  };
+
+  return (
+    <FormProvider {...form}>
+      <form onSubmit={form.handleSubmit(onSubmit)} className="w-[400px] space-y-4">
+        <div>
+          <label className="block text-sm font-medium mb-1">Username</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)